infrastructure/code/template/details.html

```diff
@@ -600,8 +950,9 @@ <h2 class="example-fixes-header">Fixed Code Examples</h2>
       };
 
       // Event listeners for main and nested tabs
+      // TODO: move this into the HTML as an onclick
       document.querySelector(MAIN_TAB_NAV_SELECTOR).addEventListener('click', onMainTabClicked);
-      document.querySelector(EXAMPLE_TAB_NAV_SELECTOR).addEventListener('click', onExampleTabClicked);
+      document.querySelector(EXAMPLE_TAB_NAV_SELECTOR)?.addEventListener('click', onExampleTabClicked);
```

Comment:
Maybe we need to go for something similar to this?

https://github.com/snyk/snyk-ls/pull/526#discussion_r1623497561